src/js/utils/dom.js

```diff
@@ -65,6 +65,12 @@ export function sanitizeElementAttributes(element) {
         if (/^on/.test(name)) {
             element.removeAttribute(name);
         }
+        if (/href/.test(name)) {
+            const link = attributes[i].value;
+            if (/javascript:|javascript&colon;/.test(link)) {
```

Comment:
this def fixes it and makes sense. are there any other ways of gettting js into a tag we could patch while we're in here? also why `javascript:|javascript&colon;` trying to save people with `javascript` in their urls i'd guess?

Reply:
Yep!